packages/@okta/vuepress-site/docs/guides/protect-your-api/main/python/reqautheverything.md

```diff
@@ -1,11 +1,30 @@
-To force a check for `login` on all routes, use `@before_request` to check route access. For example:
+To check for a successful sign-in request whenever an endpoint is requested, use `@before-request` to check for the authorization request header:
```

Comment:
This sentence has two "to check" statements; unclear which one I'm doing. Could we says something like 

Whenever an endpoint is requested, use `@before-request` to check for the authorization request header and verify that the sign-in request was successful.